src/cpp/src/llm_pipeline.cpp

```diff
@@ -523,9 +525,10 @@ ov::genai::LLMPipeline::LLMPipeline(
     const std::string& device,
     const ov::AnyMap& plugin_config
 ){
-    auto start_time = std::chrono::steady_clock::now();    
-    if ("CB" == device) {
-        m_pimpl = std::make_unique<ContinuousBatchingAdapter>(model_path, tokenizer, "CPU", plugin_config);
+    auto start_time = std::chrono::steady_clock::now();
+    if (auto scheduler_config_prop = plugin_config.find(ov::genai::enable_continuous_batching.name()) != plugin_config.end()) {
+        auto scheduler_config = plugin_config.at(ov::genai::enable_continuous_batching.name());
```

Comment:
why variable is called scheduler_config, while it's actually enable_continuous_batching ?